iphone/Classes/TiUIShortcutProxy.h

```diff
@@ -0,0 +1,34 @@
+/**
+* Appcelerator Titanium Mobile
+* Copyright (c) 2020 by Appcelerator, Inc. All Rights Reserved.
+* Licensed under the terms of the Apache Public License
+* Please see the LICENSE included with this distribution for details.
+*/
+
+#if defined(USE_TI_UISHORTCUT) || defined(USE_TI_UISHORTCUTITEM)
+
+#import <TitaniumKit/ObjcProxy.h>
+
+@class TiUIShortcutItemProxy;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol TiUIShortcutProxyExports <JSExport>
+READONLY_PROPERTY(NSArray<TiUIShortcutItemProxy *> *, items, Items);
+READONLY_PROPERTY(NSArray<TiUIShortcutItemProxy *> *, staticItems, StaticItems);
+
+- (TiUIShortcutItemProxy *)getById:(NSString *)identifier;
+
+- (void)remove:(TiUIShortcutItemProxy *)shortcut;
```

Comment:
cc @garymathews So this is me really super nit-picking. Personally I always do a double-take on any method that returns void. It offers no direct means of determining what happened, or if something happened. You need to usually poke around the object more to tell if what you wanted to happen actually did.

So in this case, we're really talking about a collection and simply adding/removing elements. I think it'd be useful to know if that actually happened. So I think perhaps in this case a boolean return type may make sense.

For example, what if I pass in a shortcut to remove that was never added in the first place? That should return false and do nothing. What if the collection was already empty, removeAll should return false. Add would likely always return true or stay void - any inability to add a shortcut should more likely be an Error thrown.

You guys don't need to change this, just something to consider. Languages/frameworks are very spotty for something like these operations where a lot return void (Obj-C NSMutableArray), some return booleans (Java List), JS returns the new length of the collection for Array. So it's not a clear consensus. (And in Java's case, `add` always returns `true`)